sdk/search/Azure.Search.Documents/api/Azure.Search.Documents.netstandard2.0.cs

```diff
@@ -584,12 +579,12 @@ public partial class GetIndexStatisticsResult
     }
     public partial class HighWaterMarkChangeDetectionPolicy : Azure.Search.Documents.Models.DataChangeDetectionPolicy
     {
-        public HighWaterMarkChangeDetectionPolicy() { }
-        public string HighWaterMarkColumnName { get { throw null; } set { } }
+        public HighWaterMarkChangeDetectionPolicy(string highWaterMarkColumnName) { }
+        public string HighWaterMarkColumnName { get { throw null; } }
     }
     public partial class ImageAnalysisSkill : Azure.Search.Documents.Models.Skill
     {
-        public ImageAnalysisSkill() { }
+        public ImageAnalysisSkill(System.Collections.Generic.IList<Azure.Search.Documents.Models.InputFieldMappingEntry> inputs, System.Collections.Generic.IList<Azure.Search.Documents.Models.OutputFieldMappingEntry> outputs) : base (default(System.Collections.Generic.IList<Azure.Search.Documents.Models.InputFieldMappingEntry>), default(System.Collections.Generic.IList<Azure.Search.Documents.Models.OutputFieldMappingEntry>)) { }
```

Comment:
Parameters should be `IEnumerable<T>` per .NET Guidelines. @KrzysztofCwalina agreed.

Reply:
Yep, still work in progress.

Reply:
https://github.com/Azure/autorest.csharp/issues/566